src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/Sort.txt

```diff
@@ -0,0 +1,557 @@
+["a","b","B","C","D","x"]
+
+>> Sort(["Hello", "hellO", "hello", "heLLo", "HELLO", "hElLo"], Value)
+["hello","hellO","heLLo","hElLo","Hello","HELLO"]
+
+>> Sort(["Hello", "hellO", "1", "%", "2", "hElLo"], Value)
+["%","1","2","hellO","hElLo","Hello"]
+
+>> Sort(["a", "b", Blank(), "x", "J", "C"], Value)
+["a","b","C","J","x",Blank()]
+
+>> Sort(["a", "b", Blank(), "x", "J", "C"], Value, SortOrder.Descending)
+["x","J","C","b","a",Blank()]
+
+// ******** BOOLEAN PARAMETERS ********
+
+>> Sort([true,false,true,false,true], Value)
+[false,false,true,true,true]
+
+>> Sort([true,false,true,false,true], Value, SortOrder.Descending)
+[true,true,true,false,false]
+
+>> Sort([true,false,Blank(),false,true], Value)
+[false,false,true,true,Blank()]
+
+>> Sort([true,false,Blank(),false,true], Value, SortOrder.Descending)
+[true,true,false,false,Blank()]
+
+// ******** DATE PARAMETERS ********
+
+>> Sort([Date(2020, 01, 05), Date(2020, 01, 01), Date(1995, 01, 01)], Value)
+[Date(1995,1,1),Date(2020,1,1),Date(2020,1,5)]
+
+>> Sort([Date(2020, 01, 05), DateValue("12/14/2012"), DateValue("01 Jan 2022")], Value)
+[Date(2012,12,14),Date(2020,1,5),Date(2022,1,1)]
+
+>> Sort([Date(2020, 01, 05), Date(2020, 01, 01), Date(1995, 01, 01)], Value, SortOrder.Descending)
+[Date(2020,1,5),Date(2020,1,1),Date(1995,1,1)]
+
+>> Sort([Date(2020, 01, 05), DateValue("12/14/2012"), Blank(), DateValue("01 Jan 2022")], Value, SortOrder.Descending)
+[Date(2022,1,1),Date(2020,1,5),Date(2012,12,14),Blank()]
+
+// ******** TIME PARAMETERS ********
+
+>> Sort([Time(12,35,55), Time(23, 01, 01), Time(5,30,50)], Value)
+["05:30:50.000","12:35:55.000","23:01:01.000"]
+
+>> Sort([Time(12,35,55), Time(23, 01, 01), Time(5,30,50)], Value, SortOrder.Descending)
+["23:01:01.000","12:35:55.000","05:30:50.000"]
+
+>> Sort([TimeValue("5:29 PM"), Blank(), Time(5,30,50)], Value)
+["05:30:50.000","17:29:00.000",Blank()]
+
+// ******** DATETIME PARAMETERS ********
+
+>> Sort([DateTimeValue("May 16, 2019 8:00:00 AM"), DateTimeValue("May 16, 2019 8:00:00 PM"), DateTimeValue("May 16, 2019 8:00:01 PM")], Value)
+[DateTime(2019,5,16,8,0,0,0),DateTime(2019,5,16,20,0,0,0),DateTime(2019,5,16,20,0,1,0)]
+
+>> Sort([DateTimeValue("May 16, 2019 8:00:00 AM"), DateTimeValue("May 16, 2019 8:00:00 PM"), DateTimeValue("May 16, 2019 8:00:01 PM")], Value, SortOrder.Descending)
+[DateTime(2019,5,16,20,0,1,0),DateTime(2019,5,16,20,0,0,0),DateTime(2019,5,16,8,0,0,0)]
+
+
+// ******** NULL and ERROR PARAMETERS ********
+
+//Erroneous sort formula
+>> Sort([-2, -1, 0, 1, 2], 1 / Value)
+#Error(Kind=Div0)
+
+>> Sort([-2, -1, 5, 1, 2], 1/0, SortOrder.Descending)
+#Error(Kind=Div0)
+
+//Error passed as one of the table records
+>> Sort([-2, -1, 5, 1/0, 2], Value)
+#Error(Kind=Div0)
+
+>> Sort(["a", "b", "D", "x", "J", Error({Kind: 11})], Value)
+#Error(Kind=Validation)
+
+>> Sort([1/0], Blank())
+#Error(Kind=Div0)
+
+// Sort error with bool value as sort formula
+>> Sort([1/0],true)
+#Error(Kind=Div0)

```

Comment:
This is not correct - the result is a valid table with a single record with an error property)

Reply:
Fixed the expected result to the correct one - Table({Value:#Error(Kind=13)})